README.md

```diff
@@ -0,0 +1,32 @@
+# feast-custom-stores-demo
+
+This is an example repo which implements a custom online store for Feast, and then uses it in a feature repo.
+
+## Installation
```

Comment:
It's a little bit confusing that the README doesn't talk about the building of the connector. What if we had one section on "Developing a connector" and one section on "Using the connector"? 

Also, a key thing that I think is missing are tests. 
* How do we know that this connector continues to work and isn't broken? Ideally we would have a simple test using a MySQL container that validates this code.
* We should show the connector developer how they can leverage the Feast test suite in order to test their connector.

Reply:
Yeah good point. Building and connecting should be top level sections here. Also I did some manual testing but that's easily scriptable into a few pytest tests.

Reply:
>  Also I did some manual testing but that's easily scriptable into a few pytest tests.

I specifically would like to know when this connector demo is broken, so minimally we need those tests, but ideally we should make it possible for contributors to run our test suite against their connector. Something like `pytest sdk/python/feast --custom-online-store class.path.to.online.OnlineStore`

Reply:
I added tests here as the github actions. I'll do a different pass to support the `custom-online-store` test like you're mentioning cause it'll be in the main feast repo.